pom.xml

```diff
@@ -187,35 +187,36 @@
       <version>1.0.1</version>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/org.xerial.snappy/snappy-java -->
```

Comment:
If we are linking to Maven, it would be good to use the official portal https://search.maven.org/ and not this third-party site with lots of ads etc :)

Reply:
Done